lib/dirac_coarse.cpp

```diff
@@ -373,6 +373,14 @@ namespace quda {
     }
   }
 
+  void DiracCoarse::prefetch(QudaFieldLocation mem_space, cudaStream_t stream) const {
+    Dirac::prefetch(mem_space, stream);
+    if (Y_d) Y_d->prefetch(mem_space, stream);
```

Comment:
Perhaps we should only fetch Yhat and not Y here?  Since Yhat is only used in the solver?

Reply:
My comment on Clover also applies here I guess.  Only prefetch what we want according to `DiracCoarse` vs `DiracCoarsePC`.